Kinvey/KinveyTests/UserTests.swift

```diff
@@ -3865,6 +3865,40 @@ extension UserTests {
         }
     }
     
+    func testLogout() {
```

Comment:
It'll be useful to test the case where the logout network call fails.

Reply:
@tejasranade additional unit test were added